running locally/docker-prestissimo/docker-compose.yml

```diff
@@ -0,0 +1,119 @@
+version: '1'
+services:
+  minio:
+    hostname: minio-server
+    image: 'minio/minio:latest'
+    container_name: minio-server
+    command: server --console-address ":9001" /minio
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    environment:
+      AWS_ACCESS_KEY_ID: minioadmin
+      AWS_SECRET_ACCESS_KEY: minioadmin
+    volumes:
+      - '~/minio/data:/minio'
+    networks:
+      presto:
+        ipv4_address: 172.19.0.2
+  hadoop:
+    hostname: hadoop-master
+    image: 'prestodb/hive3.1-hive:10'
+    container_name: hadoop-master
+    environment:
+      AWS_ACCESS_KEY_ID: minioadmin
+      AWS_SECRET_ACCESS_KEY: minioadmin
+    volumes:
+      - './hadoop/core-site.xml:/opt/hadoop/etc/hadoop/core-site.xml'
+    ports:
+    - '9083:9083'
+    depends_on:
+      - minio
+    networks:
+      presto:
+        ipv4_address: 172.19.0.3
+  coordinator:
+    container_name: coordinator
+    image: prestodb/presto:0.282
+    platform: linux/amd64
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 2G
+    ports:
+      - 8080:8080
+    volumes:
+      - ./prestissimo/coordinator/log.properties:/opt/presto-server/etc/log.properties
+      - ./prestissimo/coordinator/config.properties:/opt/presto-server/etc/config.properties
+      - ./prestissimo/coordinator/jvm.config:/opt/presto-server/etc/jvm.config
+      - ./prestissimo/coordinator/node.properties:/opt/presto-server/etc/node.properties
+      - ./prestissimo/catalog/jmx.properties:/opt/presto-server/etc/catalog/jmx.properties
+      - ./prestissimo/catalog/tpcds.properties:/opt/presto-server/etc/catalog/tpcds.properties
+      - ./prestissimo/catalog/tpch.properties:/opt/presto-server/etc/catalog/tpch.properties
+      - ./prestissimo/catalog/minio.properties:/opt/presto-server/etc/catalog/minio.properties
+    depends_on:
+      - minio
+      - hadoop
+    environment:
+      - AWS_ACCESS_KEY=FEEF05540F5D8BE7EB62
```

Comment:
is this the release value?

Reply:
I could change this to be like this https://github.com/prestodb/prestorials/blob/main/docker-compose/docker-compose.yaml:
```
environment:
      - AWS_ACCESS_KEY=${AWS_ACCESS_KEY}
      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
```

What do you think?